spec/integration/lib/auth0/api/v2/api_tickets_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'spec_helper'
+describe Auth0::Api::V2::Tickets do
+  attr_reader :client, :user
+
+  before(:all) do
+    @client = Auth0Client.new(v2_creds)
+    username = Faker::Internet.user_name
+    email = "#{entity_suffix}#{Faker::Internet.safe_email(username)}"
+    password = Faker::Internet.password
+    connection = 'Username-Password-Authentication'
```

Comment:
I see "Username-Password-Authentication" string spread all over the project. Should be create a global constant?